lib/creditsafe/client.rb

```diff
@@ -86,6 +91,15 @@ def invoke_soap(message_type, message)
       notification_payload[:response] = response.body
     rescue => raw_error
       processed_error = handle_error(raw_error)
+
+      if processed_error.class == AccountError
+        retries -= 1
+        if retries.positive?
+          sleep(RETRY_SLEEP_TIME)
```

Comment:
Good point here. Is it worth retrying a few times at least with a shorter (ideally sub-1s) pause? How clustered are these failures?